README.md

```diff
@@ -156,7 +156,7 @@ Execute the binary, with `-h` to see the built-in help, e.g. `./Fulcrum -h`. You
 
 It is recommended you specify a data dir (`-D` via CLI or `datadir=` via config file) on an SSD drive for best results.  Synching against `testnet` should take you about 10-20 minutes (more on slower machines), and mainnet can take anywhere from 4 hours to 20+ hours, depending on machine and drive speed.  I have not tried synching against mainnet on an HDD and it will probably take ***days*** if you are lucky.
 
-Once the server finishes synching it will behave like an ElectronX/ElectrumX server and it can receive requests from Electron Cash (or Electrum if on BTC).
+As long as the Fulcrum server is still synchronizing, an attempt to communicate with the server will fail with the error: "Connection refused". Once the server finished synching it will behave like an ElectronX/ElectrumX server and it can receive requests from Electron Cash (or Electrum if on BTC).
```

Comment:
Two things::

1. s/finished/finishes
2. As @jonas-lundqvist pointed out in slack, the change implies that nothing works, but the HTTP /stats endpoint does work.
 
```suggestion
As long as the Fulcrum server is still synchronizing, an attempt to communicate with the server's RPC ports will fail with the error: "Connection refused". Once the server finishes synching it will behave like an ElectronX/ElectrumX server and it can receive requests from wallet clients such as Electron Cash (or Electrum, Sparrow, etc if on BTC).  Note that the HTTP "stats" and "debug" endpoints *do* work even while synching (the HTTP server endpoint is specified with `-z` on the CLI or `stats=` in the config file). 
```

Reply:
`/stats` end also doesn't work while synching. 
![image](https://user-images.githubusercontent.com/628926/206010039-8240e037-3f2f-423b-8c3b-58fc13d2d8e4.png)

Reply:
Only **AFTER** the Sync is complete, I can use the `/stats` command. As you can see above, I couldn't use the stats during synching.

![image](https://user-images.githubusercontent.com/628926/206025281-8a6c90fb-3ba2-41a9-b5fc-280b342f4d30.png)

Reply:
uh.. lol..?? 😕  So uh, maybe it doesn't work as I would love to see it?

Reply:
I mean it "Works" in that the port is there, the HTTP server is there.. just that /stats is super low-priority.  Idk maybe not worth mentioning this in the readme at all..

Maybe your language is preferable since it doesn't really "work reliably".. hmm.

Reply:
Or I guess some people might assume that all the ports would be there but they should return error messages.. (which is what bitcoind does for HTTP RPC)... hmm.

Ok yeah the section as you have it now is fine, I guess...

Reply:
I mean but why would you assume that a server that is designed to be public-facing would open its public-facing ports to the world in a totally not-ready-to-do-business state??  Is that good software design? 

Anyway yeah we can leave that section ... I guess others might assume the same thing and worry... if you did others will too.

Reply:
Well.. does it help if I say it's my birthday? Since it is.

Reply:
Ha ha ha Happy Birthday man!

Reply:
Thanks 😄 !